meta-dts-distro/recipes-dts/dts/dasharo-deploy/dasharo-deploy

```diff
@@ -104,13 +106,28 @@ SERIAL_NUMBER_DIR="/tmp/serial_number.txt"
 PROGRAMMER_BIOS="internal"
 PROGRAMMER_EC="ite_ec"
 
+DASHARO_VENDOR="3mdeb"
```

Comment:
@esesyn01 please add here some comment why we need those variables

Reply:
@TomaszAIR I added it there - https://github.com/Dasharo/meta-dts/pull/25/commits/895a9ce7e8a164f6febc0a4e3958214e5000e7cc#diff-08011e9585e945f2174ca7dd5572e8e0da2352a51eb05ca60b75a2475b834645R9